decide-app/src/test/java/com/decide/app/LaunchInterceptorConditionsTest.java

```diff
@@ -102,7 +102,103 @@ public void LIC0ThrowsIllegalArgumentExceptionOnInvalidParameter() {
         );
     }
   
-  
+
+    /**
+     * ========================== [ LIC 3 ] ==========================
+     */
+
+    /**
+     * Positive test case, ensure LIC3 is satisfied when three consecutive
+     * points are the vertices of a triangle with area greater than AREA1.
+     */
+    @Test
+    public void LIC3TrueOnAreaGreaterThanAREA1() {
+        int NUMPOINTS = 3;
+        Parameters PARAMETERS = new Parameters();
+        PARAMETERS.AREA1 = 1.9;
+        Point[] POINTS = new Point[]{
+            new Point(0,0),
+            new Point(2,0),
+            new Point(0,2)
+        };
+        LaunchInterceptorConditions LIC = new LaunchInterceptorConditions(NUMPOINTS, POINTS, PARAMETERS);
+        assertTrue(LIC.getLaunchInterceptorCondition3());
+    }
+
+    /**
+     * Negative test case, ensure LIC3 is not satisfied when three consecutive
+     * points are the vertices of a triangle with area lower than AREA1
+     */
+
+     @Test
+     public void LIC3FalseOnAreaLowerThanAREA1() {
+         int NUMPOINTS = 3;
+         Parameters PARAMETERS = new Parameters();
+         PARAMETERS.AREA1 = 2.1;
+         Point[] POINTS = new Point[]{
+             new Point(0,0),
+             new Point(2,0),
+             new Point(0,2)
+         };
+         LaunchInterceptorConditions LIC = new LaunchInterceptorConditions(NUMPOINTS, POINTS, PARAMETERS);
+         assertFalse(LIC.getLaunchInterceptorCondition3());
+     }
+
+    /** * Edge-case test case, ensure LIC3 is not satisfied when three consecutive
+     * points are the vertices of a triangle with the exact same AREA1
+     */
+
+     @Test
+     public void LIC3FalseOnLessThanThreePoints() {
+         int NUMPOINTS = 2;
+         Parameters PARAMETERS = new Parameters();
+         PARAMETERS.AREA1 = 2;
+         Point[] POINTS = new Point[]{
+             new Point(0,0),
+             new Point(2,0)
+         };
+         LaunchInterceptorConditions LIC = new LaunchInterceptorConditions(NUMPOINTS, POINTS, PARAMETERS);
+         assertFalse(LIC.getLaunchInterceptorCondition3());
+     }
+
+    /**
+     * Invalid input test case, ensure LIC3 throws IllegalArgumentException
+     * if the supplied parameter AREA1 is less than 0.
+     */
+    @Test
+    public void LIC3ThrowsIllegalArgumentExceptionOnInvalidAREA1() {
+        int NUMPOINTS = 1;

```

Comment:
There are three points, please set NUMPOINTS to be `int NUMPOINTS = POINTS.length` for added clarity and less risk of bugs